src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -41,7 +44,7 @@ public function process(ContainerBuilder $container)
         try {
             $this->container = $container;
             foreach ($container->getDefinitions() as $id => $definition) {
-                if (false !== $autowired = $definition->getAutowired()) {
+                if (!empty($autowired = $definition->getAutowired())) {
```

Comment:
I'm not a bug fan of using `empty`. I'd prefer `if ($autowired = $definition->getAutowired()) {`